internal/worker/api/errors.go

```diff
@@ -0,0 +1,181 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+const (
+	ErrorCodePrefix = "COMPOSER-WORKER-"
+	ErrorHREF       = "/api/composer-worker/v1/errors"
+
+	ErrorUnsupportedMediaType ServiceErrorCode = 3
+	ErrorJobNotFound          ServiceErrorCode = 5
+	ErrorJobNotRunning        ServiceErrorCode = 6
+	ErrorMalformedJobId       ServiceErrorCode = 7
+	ErrorMalformedJobToken    ServiceErrorCode = 8
+	ErrorInvalidErrorId       ServiceErrorCode = 9
+	ErrorBodyDecodingError    ServiceErrorCode = 10
+	ErrorResourceNotFound     ServiceErrorCode = 11
+	ErrorMethodNotAllowed     ServiceErrorCode = 12
+	ErrorNotAcceptable        ServiceErrorCode = 13
+	ErrorErrorNotFound        ServiceErrorCode = 14
+	// ErrorTokenNotFound ServiceErrorCode = 6
```

Comment:
Oh woops, yea I think I ended up not using ErrorTokenNotFound, but I think it would be useful as a return code for https://github.com/osbuild/osbuild-composer/pull/1784#discussion_r713229537  (and it should be = 15 then yea).

But I don't wanna change the behaviour in this PR so let's just not?

Reply:
Sure!